contracts/payloads/IGP47/PayloadIGP47.sol

```diff
@@ -0,0 +1,773 @@
+        TYPE vaultType;
+        address supplyToken;
+        address borrowToken;
+    }
+
+    constructor() {
+        ADDRESS_THIS = address(this);
+    }
+
+    function propose(string memory description) external {
+        require(
+            msg.sender == PROPOSER ||
+                msg.sender == TEAM_MULTISIG ||
+                address(this) == PROPOSER_AVO_MULTISIG ||
+                address(this) == PROPOSER_AVO_MULTISIG_2 ||
+                address(PROPOSER_AVO_MULTISIG_3) == PROPOSER_AVO_MULTISIG_3,
+            "msg.sender-not-allowed"
+        );
+
+        uint256 totalActions = 1;
+        address[] memory targets = new address[](totalActions);
+        uint256[] memory values = new uint256[](totalActions);
+        string[] memory signatures = new string[](totalActions);
+        bytes[] memory calldatas = new bytes[](totalActions);
+
+        // Action 1: call executePayload on timelock contract to execute payload related to Fluid and Lite
+        targets[0] = address(TIMELOCK);
+        values[0] = 0;
+        signatures[0] = "executePayload(address,string,bytes)";
+        calldatas[0] = abi.encode(ADDRESS_THIS, "execute()", abi.encode());
+
+        uint256 proposedId = GOVERNOR.propose(
+            targets,
+            values,
+            signatures,
+            calldatas,
+            description
+        );
+
+        require(proposedId == PROPOSAL_ID, "PROPOSAL_IS_NOT_SAME");
+    }
+
+    function execute() external {
+        require(address(this) == address(TIMELOCK), "not-valid-caller");
+
+        // Action 1: Set GHO token config and market rate curve on liquidity.
+        action1();
+
+        // Action 2: Set GHO based vaults limits.
+        action2();
+    }
+
+    function verifyProposal() external view {}
+
+    /**
+     * |
+     * |     Proposal Payload Actions      |
+     * |__________________________________
+     */
+
+    /// @notice Action 1: Set GHO token config and market rate curve on liquidity.
+    function action1() internal {
+        {
+            AdminModuleStructs.RateDataV1Params[]
+                memory params_ = new AdminModuleStructs.RateDataV1Params[](1);
+
+            params_[0] = AdminModuleStructs.RateDataV1Params({
+                token: GHO_ADDRESS, // GHO
+                kink: 93 * 1e2, // 93%
+                rateAtUtilizationZero: 0, // 0%
+                rateAtUtilizationKink: 7.5 * 1e2, // 7.5%
+                rateAtUtilizationMax: 100 * 1e2 // 100%
+            });
+
+            LIQUIDITY.updateRateDataV1s(params_);
+        }
+    }
+
+    /// @notice Action 2: Set GHO based vaults limits.
+    function action2() internal {
+        VaultConfig memory vaultConfig = VaultConfig({
+            vaultId: 0,
+            supplyToken: address(0),
+            supplyMode: 1, // Mode 1
+            supplyExpandPercent: 25 * 1e2, // 25%
+            supplyExpandDuration: 12 hours, // 12 hours
+            supplyBaseLimitInUSD: 5_000_000, // $5M
```

Comment:
@KaymasJain should this be 7.5M? we only had 5M for CBBTC vaults initially at launch

Reply:
Yes, 7.5M.